### DISCUSSION TITLE
Express the results in `sympy.core.relational.StrictLessThan`

### DISCUSSION THREAD

**Discussion by 1e9abhi1e10** [sympy/sympy#24655] [Q&A] [answered]
In solving #24288 , for the tests `assert parse_expr('1 < 2', evaluate=False) == result`
where the result type is `sympy.core.relational.StrictLessThan`
The result I got on my branch
```python
>>> parse_expr('1 < 2', evaluate=False)  
1 < 2
>>> type(parse_expr('1 < 2', evaluate=False))
<class 'sympy.core.relational.StrictLessThan'>
```
But Lt(1, 2) is boolean
but when I used non-numeric values I got the desired results
```python
>>> x,y = symbols('x y')
>>> type(Lt(x,y))
<class 'sympy.core.relational.StrictLessThan'>
```
If am not use the above results then how can I express the result in <class 'sympy.core.relational.StrictLessThan'>
Is their a way to express the results in sympy.core.relational.StrictLessThan'> in that form ?

**Comment by oscarbenjamin** [comment, marked as answer]
You can use `evaluate=False`:
```python
In [1]: Lt(1, 2, evaluate=False)
Out[1]: 1 < 2
```

**Reply by 1e9abhi1e10** [reply]
Okay , thanks a lot!
